### ISSUE TITLE
Shortcut is it working as expected

### ISSUE THREAD

**Issue by DavideRossi** [atareao/Touchpad-Indicator#19]
It seems like the keyboard shortcut can only be used to disable the touchpad (at least this is how it works for me).
If this is the intended behavior would it be possible to change that to a toggle so that the touchpad can be activated if currently not active?
Sometimes I have issues with my Bluetooth mouse when resuming from sleep mode and I'm stuck with the mouse not working and the touchpad disabled by touchpad-Indicator...

**Comment by atareao** [comment]
No. The shortcut must enable and disable the touchpad. Please try another shortcat and tell me.

**Comment by Jack-Garbo** [comment]
Installed TI 2.2.1 on xubuntu 18.04. Works, except for essential shortcut. It's not on the menu. has it been dropped?
